examples/hubert/lightning.py

```diff
@@ -0,0 +1,133 @@
+from typing import Tuple
+
+import torch
+import torchaudio
+from dataset import BucketizeSampler, DistributedBatchSampler, HuBERTDataSet, CollateFnHubert
```

Comment:
BucketizeSampler not defined.

Reply:
Hi @kobenaxie, thanks for pointing it out. I recently updated the `BucketizeSampler` to `BucketizeBatchSampler` in https://github.com/pytorch/audio/blob/main/examples/hubert/dataset/hubert_dataset.py#L11. I will update this PR soon.